_saas-integrations/workday/v1/workday-raas-v1.md

```diff
@@ -0,0 +1,133 @@
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Workday RaaS
+permalink: /integrations/saas/workday-raas
+keywords: workday-raas, integration, schema, etl workday-raas, workday-raas etl, workday-raas schema
+layout: singer
+# input: false
+
+key: "workday-raas-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "workday-raas"
+display_name: "Workday RaaS"
+
+singer: true
+status-url: ""
+
+tap-name: "workday-raas"
+repo-url: https://github.com/singer-io/tap-workday-raas
+
+this-version: "1"
+
+api: |
+  [SOAP API](https://community.workday.com/api){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: true
+
+api-type: "platform.workday-raas" 
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Enterprise"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. There are no schema files for this integration.
+
+
+# -------------------------- #
+#      Incompatibilities     #
+# -------------------------- #
+
+## uncomment section below if integration is compatible with any Stitch destinations
+## if incompatible with multiple destinations, create a section for each destination
+
+## incompatible:
+  ## [redshift]: "always,sometimes,never"
+  ## reason: "copy" 
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+setup-steps:
+  - title: "Retrieve your report URL."
+    anchor: "retrieve-report-url"
+    content: |
+      1. Login to your {{ integration.display_name }} account.
+      2. Navigate to your reports.
+      3. Select the report you want to replicate in Stitch and ensure that the **Enable As Web Service** box is checked.
+      4. Copy the report URL and keep it available for the next step.
+
+      {% include note.html type=single-line content="**Note**: If you would like to replicate multiple reports, repeat steps 3 & 4." %}
+
+  - title: "add integration"
+    content: |
+      4. In the **Username** field, enter the username for your {{ integration.display_name }} account.
+      5. In the **Password** field, enter the password for your {{ integration.display_ name }} account.
+      6. In the **Report** field, enter the name of your report.
+      7. In the **Report** field, enter the report URL you obtained in [step 1](#retrieve-report-url).
+
+      {% include note.html type=single-line content="**Note**: If you would like to replicate more reports, click on **Configure another report**." %}
```

Comment:
Same here as my other comment about callouts - I think you can just have this as a separate sentence instead of a callout.